common/src/test/java/com/microsoft/hydralab/common/performance/PerformanceTestManagementServiceTest.java

```diff
@@ -18,4 +20,16 @@ public void testInspect_ReturnNull() {
     public void testParse_ReturnNull() {
         Assertions.assertNull(performanceTestManagementService.parse(null));
     }
+
+    @Test
+    public void testInspectEvent_ReturnNull() {
+        PerformanceInspection eventStartInspection = PerformanceInspection.createEventStartInspection("", "event start");
```

Comment:
```suggestion
        String deviceIdentifier = "";
        PerformanceInspection eventStartInspection = PerformanceInspection.createEventStartInspection(deviceIdentifier, "event start");
```
Give the param a name to clarify the meaning.

Reply:
Maybe provide a single param creation method.